sdk/core/azure-core/inc/azure/core/http/policy.hpp

```diff
@@ -273,14 +273,14 @@ namespace Azure { namespace Core { namespace Http {
     }
 
     std::unique_ptr<RawResponse> Send(
-        Context const& ctx,
         Request& request,
-        NextHttpPolicy nextHttpPolicy) const override
+        NextHttpPolicy nextHttpPolicy,
+        Context const& ctx) const override
     {
       auto uuid = Uuid::CreateUuid().ToString();
 
       request.SetHeader(RequestIdHeader, uuid);
-      return nextHttpPolicy.Send(ctx, request);
+      return nextHttpPolicy.Send(request, ctx);
```

Comment:
nit: I've seen that `ctx` from the old times we started the repo, we started to use `context` later on. This might be a good opportunity to update all occurrences.